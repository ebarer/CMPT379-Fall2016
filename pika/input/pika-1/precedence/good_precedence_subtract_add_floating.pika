exec {

  print 1.5 - 9.3 + 5.4, _n_.
  print 7.0 - 1.9 + 6.9, _n_.
  print 3.3 - 9.0 + 9.4, _n_.
  print 7.7 - 2.9 + 6.7, _n_.
  print 6.8 - 9.4 + 3.3, _n_.
  print 6.3 - 5.1 + 7.2, _n_.
  print 2.7 - 2.8 + 5.8, _n_.
  print 4.5 - 5.5 + 3.7, _n_.
  print 8.8 - 6.5 + 8.7, _n_.
  print 8.2 - 6.2 + 3.2, _n_.
  print 7.3 - 3.9 + 4.5, _n_.
  print 5.4 - 1.0 + 9.8, _n_.

  print 8.0 + 2.7 - 8.8, _n_.
  print 7.8 + 6.5 - 2.6, _n_.
  print 8.9 + 5.8 - 5.5, _n_.
  print 1.8 + 2.1 - 3.1, _n_.
  print 5.1 + 3.9 - 4.2, _n_.
  print 6.4 + 5.0 - 7.0, _n_.
  print 7.0 + 6.0 - 2.6, _n_.
  print 6.9 + 5.1 - 9.5, _n_.
  print 8.4 + 7.5 - 7.8, _n_.
  print 3.7 + 7.8 - 5.4, _n_.
  print 2.8 + 4.6 - 3.4, _n_.
  print 8.4 + 9.8 - 9.8, _n_.

  print 7.1 - 2.6 + 5.9 - 2.1, _n_.
  print 4.8 - 3.3 + 9.6 - 8.8, _n_.
  print 4.1 - 5.5 + 2.2 - 5.8, _n_.
  print 2.5 - 8.6 + 9.2 - 6.4, _n_.
  print 1.9 - 6.0 + 1.7 - 3.9, _n_.
  print 8.1 - 7.8 + 2.1 - 3.8, _n_.
  print 7.8 - 6.3 + 7.6 - 2.9, _n_.
  print 7.6 - 8.3 + 2.8 - 9.3, _n_.
  print 4.4 - 8.3 + 9.7 - 4.8, _n_.
  print 2.8 - 9.1 + 7.8 - 6.4, _n_.
  print 6.2 - 3.3 + 6.3 - 6.9, _n_.
  print 7.5 - 4.4 + 8.3 - 5.3, _n_.

  print 2.8 + 2.0 - 4.2 + 8.8, _n_.
  print 1.7 + 2.8 - 8.1 + 3.6, _n_.
  print 1.9 + 8.6 - 8.5 + 8.3, _n_.
  print 7.9 + 3.8 - 3.0 + 8.2, _n_.
  print 5.7 + 1.8 - 1.9 + 2.0, _n_.
  print 1.4 + 4.1 - 3.5 + 3.1, _n_.
  print 6.9 + 3.3 - 7.0 + 8.7, _n_.
  print 5.8 + 5.8 - 8.9 + 1.9, _n_.
  print 5.2 + 6.0 - 7.5 + 2.7, _n_.
  print 8.1 + 4.5 - 8.6 + 1.1, _n_.
  print 6.0 + 2.3 - 3.5 + 2.5, _n_.
  print 1.8 + 6.9 - 1.4 + 3.4, _n_.

}

