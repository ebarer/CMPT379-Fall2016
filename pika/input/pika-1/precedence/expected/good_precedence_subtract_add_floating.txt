-2.4
12
3.7
11.5
0.7
8.4
5.7
2.7
11
5.2
7.9
14.2
1.9
11.7
9.2
0.8
4.8
4.4
10.4
2.5
8.1
6.1
4
8.4
8.3
2.3
-5
-3.3
-6.3
-1.4
6.2
-7.2
1
-4.9
2.3
6.1
9.4
4.44089e-016
10.3
16.9
7.6
5.1
11.9
4.6
6.4
5.1
7.3
10.7
